workflow/metrics/server.go

```diff
@@ -14,9 +15,20 @@ import (
 	runtimeutil "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/utils/env"
 
+	"github.com/argoproj/argo-workflows/v3"
 	tlsutils "github.com/argoproj/argo-workflows/v3/util/tls"
 )
 
+var (
+	buildInfo = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "build_info",
+			Help: "A metric with a constant '1' value labeled by version from which Argo-Workflows was built.",
```

Comment:
and is not referred to as "Argo-Workflows" within the codebase or docs (generally speaking). 

this should also be consistent with the comment

```suggestion
			Help: "A metric with a constant '1' value labeled with metadata about Argo's version and runtime.",
```